package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think package.json file in the root folder is of any use. Same goes for the node_modules

Reply:
No. I think we need the gitignore file with those. Let me change it.

Reply:
I think, there is another package.json directly in the root. There is one that is in the `frontend` folder which is fine, but I can't see why we need `package.json` in the root folder, outside of the `frontend` folder.